ubuntudesign/vanilla-framework
# Cleanup ‘.row’

## Done

The class `.row` should be used purely as a layout construct to contain columns of the grid. This changeset removed the verbose styling for `.row` plus related variants and moves the now lean `.row` class into with grid styling.

Note: Some of the styling for light/dark backgrounds which has previously been attached to row variants will be used on the new pattern `.strip` in due course.

## QA

Run code, check that `.row` behaves as described in code and as in this demo: http://codepen.io/barrymcgee/pen/qNLNVV

## Details

Fixes: #409
